lib/ReactGridLayout.jsx

```diff
@@ -721,7 +721,7 @@ export default class ReactGridLayout extends React.Component<Props, State> {
     // to avoid unpredictable jumping of a dropping placeholder
     if (
       isFirefox &&
-      !e.nativeEvent.target.className.includes(layoutClassName)
+      !(new RegExp(layoutClassName).test(e.nativeEvent.target.className))
```

Comment:
```suggestion
      e.nativeEvent.target.className.indexOf(layoutClassName) === -1
```

Reply:
I agree about the readability and I've done the changes. Should be fine now.